.golangci.yml

```diff
@@ -1,11 +1,25 @@
 linters:
   enable-all: true
   disable:
+    - cyclop
+    - exhaustivestruct  # TODO: activate
+    - forbidigo
     - gochecknoglobals # Cobra requires globals
     - gochecknoinits # Cobra requires init blocks
     - goconst # tests contain a ton of hard-coded test strings, for example branch names
+    - gofmt   # TODO: activate
+    - gofumpt   # TODO: activate
+    - goimports  # TODO: activate
+    - golint
     - gomnd # tests contain hard-coded test data that wouldn't make sense to extract into constants
+    - ifshort
+    - interfacer
+    - ireturn
     - lll # we aren't enforcing a line length at this point
+    - maligned  # deprecated
```

Comment:
golangci-lint prints warnings that these linters are deprecated, hence I disable them here. The goal is a clean linter output without any errors or warnings.